CustomLocalePlugin.inc.php

```diff
@@ -20,40 +20,44 @@ class CustomLocalePlugin extends GenericPlugin {
 	 * @param $category string Plugin category
 	 * @param $path string Plugin path
 	 */
-	function register($category, $path) {
-		if (parent::register($category, $path)) {
-			if ($this->getEnabled()) {
-
-				// Add custom locale data for already registered locale files.
-				$locale = AppLocale::getLocale();
-				$localeFiles = AppLocale::getLocaleFiles($locale);
-
-				$context = Request::getContext();
-				$contextId = $context->getId();
-
-				$publicFilesDir = Config::getVar('files', 'public_files_dir');
-				$customLocalePathBase = "$publicFilesDir/presses/$contextId/" . CUSTOM_LOCALE_DIR . "/$locale/";
-
-				import('lib.pkp.classes.file.FileManager');
-				$fileManager = new FileManager();
-				foreach ($localeFiles as $localeFile) {
-					$customLocalePath = $customLocalePathBase . $localeFile->getFilename();
-					if ($fileManager->fileExists($customLocalePath)) {
-						AppLocale::registerLocaleFile($locale, $customLocalePath, false);
-					}
+	/* UZH CHANGE OJS-74 2019/03/05/mb adapt to OJS 3.1.2 */
+	function register($category, $path, $mainContextId = null) {
```

Comment:
Moved the warning message resolution into a separate commit: https://github.com/asmecher/customLocale/commit/7a5cfcdf29447950c5fb5bab969937a16ce9cd5e